src/stylesheets/components/_alerts.scss

```diff
@@ -100,6 +91,8 @@ $padding-left: units($theme-alert-padding-x) + units($theme-alert-bar-width);
 
 .usa-alert-text {
   @include u-margin-y(0);
+
+  a { @include typeset-link; }
```

Comment:
Can this be written as:
```
a { 
  @include typeset-link; 
}
```
to be consistent with the rest of the code base?

Reply:
yeah, will do